src/main/java/fr/emse/majeureinfo/springbootintro/web/RobotDto.java

```diff
@@ -20,12 +20,12 @@ public Long getId() {
         return id;
     }
 
-    public Sensor getSensor() {
-        return sensor;
+    public SensorDto getSensor() {
+        return new SensorDto(sensor);
```

Comment:
good but you've forgotten the attributs of the class ;-(

Reply:
I don't even understand how I missed such basic stuff... :(

Reply:
by the way, because of this error, you only have 19/20 ;-)

Reply:
What a pity! I am so disappointed! ;)